src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/Listing.html

```diff
@@ -1,3 +1,6 @@
+<div class="overview-description">
+    <adh-parse-markdown data-parsetext="overviewDesc"></adh-parse-markdown>
+</div>
```

Comment:
@xi and I agree there should be an overview directive that wraps `adh-process-listing` and that the text should be placed there.